MAUI/Calendar/date-navigation.md

```diff
@@ -70,6 +70,29 @@ this.Calendar.AllowViewNavigation = true;
 {% endhighlight %}
 {% endtabs %}
 
+## Navigation to Adjacent Months Using Leading/Trailing Dates
```

Comment:
Navigating Adjacent Months with Leading and Trailing Dates

Reply:
Changed it